tests/functional/firefox/test_home.py

```diff
@@ -0,0 +1,30 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.firefox.home import FirefoxHomePage
+
+
+@pytest.mark.sanity
```

Comment:
Remove this line to not run the test in IE9

Reply:
TY!